Replace deprecated -pa option in newer versions of RepeatModeler with -threads
author: Alex Domingo (Vrije Universiteit Brussel)
--- run_repeat_modeler.py.orig	2023-03-17 09:23:21.857374000 +0100
+++ run_repeat_modeler.py	2023-03-17 09:24:17.655942000 +0100
@@ -73,7 +73,7 @@
 
     # BuildDatabase -name Choanephora_cucurbitarum
     # ../Choanephora_cucurbitarum_assembly.fna
-    # RepeatModeler -database Choanephora_cucurbitarum -pa 25
+    # RepeatModeler -database Choanephora_cucurbitarum -threads 25
 
     # Get repeat model
     repeat_lib = os.path.join(output_dir, '*', 'consensi.fa.classified')
@@ -89,7 +89,7 @@
         os.system(command1)
 
         log_file2 = os.path.join(log_dir, 'repeat_modeler.log')
-        command2 = '{} -database {} -pa {} > {} 2>&1'.format(
+        command2 = '{} -database {} -threads {} > {} 2>&1'.format(
             repeatmodeler_bin, genome_assembly, num_cores, log_file2
         )
         logger_txt.debug('[Run] %s', command2)
